### DISCUSSION TITLE
Chatbot integration to instant messaging apps (e.g. Whatsapp, Telegram)

### DISCUSSION THREAD

**Discussion by hayschan** [danny-avila/LibreChat#3549] [Feature Requests & Suggestions]
By integrating LibreChat with instant messaging (IM) apps, we can make chat bots on the platforms easily.
- Telegram offers versatile API.
- WhatsApp does not offer API in its normal app. However, it offers API to WhatsApp Business account.

**Comment by berry-13** [comment]
yea would be cool to add something like. 

The easiest integrations are:
- Telegram
- Messenger
- Instagram
- Discord

**Reply by hayschan** [reply]
I have checked other projects about integration between LLM and instant messaging apps. They are all very cumbersome. LibreChat has done a very good job between the balance between versatile and easy to setup using config file.

**Comment by alvieirajr** [comment]
I'm working on It, a whatsapp front interface to use Librechat backend.

**Comment by Kathrininfanta** [comment]
Do you want to build like WhatsApp? Check this https://github.com/MirrorFly/MirrorFly-iOS-Sample  MirrorFly provides a complete set of Instant Messaging APIs and SDKs that enable you to integrate chat, voice, and video calling features into your own application. Whether you’re developing for iOS or other platforms, MirrorFly’s APIs help you create a secure, scalable, and customizable messaging experience without building everything from scratch.
